File: backend/src/main/java/com/tanerkaynar/nexuserp/model/Personel.java
package com.tanerkaynar.nexuserp.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "personeller")
public class Personel {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "personelid")
    private Integer personelid;

    @Column(name = "adsoyad", nullable = false)
    private String adsoyad;

    @Column(name = "departman")
    private String departman;

    @Column(name = "aktifmi", nullable = false)
    private Boolean aktifmi = true;

    @Column(name = "kayittarihi", insertable = false, updatable = false)
    private LocalDateTime kayittarihi;
}

File: client-desktop/NexusERP/Models/Personel.cs
using System;

namespace NexusERP.Models
{
    public class Personel
    {
        public int PersonelID { get; set; }
        public string AdSoyad { get; set; }
        public string Departman { get; set; }
        public bool AktifMi { get; set; }
        public DateTime KayitTarihi { get; set; }
    }

}